### DISCUSSION TITLE
[STAFF] Como funcionará as ideias de Natal

### DISCUSSION THREAD

**Discussion by austv** [leogianfagna/IssueTracker#995] [Perguntas] [answered]
Qual será o gerenciamento, programação e quem vai decidir as ideias de Natal para esse ano e os próximos.

**Comment by leogianfagna** [comment, marked as answer]
Dado **recente reunião sobre mudanças** tomadas pela austv com os administradores, a forma de levar eventos sazonais será alterada.

Os passos acontecem de tal forma:
1. A STAFF que decide qual será o tema do próximo evento sazonal.
2. É aberto o canal de sugestões de ideias para toda a comunidade, podendo sugerir infinitas possibilidades.
3. STAFF se reúne para **aprovar** ideias sugeridas (incluindo de ajudantes), **definindo datas** e prazos de desenvolvimento.

A partir do ponto 3, com prazos e ideias definidas, os criadores de conteúdo já poderão saber o que está por vir. As datas definidas são oficiais e os jogadores poderão saber quando terá cada coisa.
